apps/codecov-api/graphql_api/types/test_analytics/test_analytics.py

```diff
@@ -362,52 +368,56 @@ async def resolve_test_results(
     before: str | None = None,
 ) -> TestResultConnection:
     if await should_use_new_test_analytics(repository, info):
-        measurement_interval = (
-            filters.get("interval", MeasurementInterval.INTERVAL_30_DAY)
-            if filters
-            else MeasurementInterval.INTERVAL_30_DAY
-        )
-        end_date = datetime.now(UTC)
-        start_date = end_date - timedelta(days=measurement_interval.value)
-        ordering_param = (
-            ordering.get("parameter", TestResultsOrderingParameter.AVG_DURATION)
-            if ordering
-            else TestResultsOrderingParameter.AVG_DURATION
-        )
-        ordering_direction = (
-            ordering.get("direction", OrderingDirection.DESC)
-            if ordering
-            else OrderingDirection.DESC
-        )
+        with test_results_histogram.time():
+            measurement_interval = (
+                filters.get("interval", MeasurementInterval.INTERVAL_30_DAY)
```

Comment:
Nit: the double mention of `MeasurementInterval.INTERVAL_30_DAY` feels a bit odd. I'd do

`filters.get("interval") if filters else MeasurementInterval.INTERVAL_30_DAY`

If you need to check `filters`